src/mixins/pickerMixin.vue

```diff
@@ -71,10 +75,17 @@ export default {
      * @param {Number} incrementBy
      */
     changePage(incrementBy) {
-      const date = this.pageDate
-      this.utils.setFullYear(date, this.utils.getFullYear(date) + incrementBy)
+      const { pageDate, utils } = this
+      const units =
+        this.view === 'year' ? incrementBy * this.yearRange : incrementBy
+
+      if (this.view === 'day') {
+        utils.setMonth(pageDate, utils.getMonth(pageDate) + units)
+      } else {
+        utils.setFullYear(pageDate, utils.getFullYear(pageDate) + units)
+      }
 
-      this.$emit('page-change', date)
+      this.$emit('page-change', pageDate)
```

Comment:
That's true, thanks. I'll have a look at this later this evening...

Reply:
OK - I've added a quick fix for this in a new commit. Once we migrate across from `span`s to `button`s, we won't need this as I'll add e.g. a `:disabled=isPreviousDisabled` attribute to the button.

While we're on this subject, can I ask your opinion about whether it will be necessary to keep the `.disabled` class in addition to the `:disabled` attribute? It seems like duplication to me, but I guess some people might currently be targeting their styles based on `span.disabled`? Do we ask them to migrate to `button:disabled` ?

Reply:
If we migrate from `span` to buttons we need a major version change anyway. Some people are selecting with the HTML tag. Which means we can change this too.

Reply:
Ok thanks.